install_config/install/docker_registry.adoc

```diff
@@ -169,9 +170,68 @@ $ oadm registry --service-account=registry \
 ----
 endif::[]
 
-[[viewing-logs]]
 
+[[registry-ip-address]]
+=== Registry IP Address
```

Comment:
@bfallonf 
> Can this have a more descriptive title? Something like "Defining the Registry IP Address" maybe? Something that describes the process the reader is about to undertake.
IIUC the procedure is only necessary if you want to remove and then re-add the `docker-registry` service.  The purpose of the procedure is to maintain the IP address, so perhaps a better title might be "Maintaining the Registry IP Address".